samcli/settings/__init__.py

```diff
@@ -17,7 +17,7 @@
 
 import os
 
-if "__SAM_CLI_TELEMETRY_ENDPOINT_URL" not in os.environ:
-    telemetry_endpoint_url = "https://aws-serverless-tools-telemetry.us-west-2.amazonaws.com/metrics"
-else:
-    telemetry_endpoint_url = os.getenv("__SAM_CLI_TELEMETRY_ENDPOINT_URL")
+
+telemetry_endpoint_url = os.getenv(
```

Comment:
There is a reason why this was coded this way, just racking my brain to remember it.

Reply:
But according to the [python doc](https://docs.python.org/3/library/os.html#os.getenv), this should be same effect.

Reply:
seems alright.